vm e2e tests failure on Release 1.11.3(https://github.com/antrea-io/antrea/pull/5239)

**Describe the bug**
<!--
A clear and concise description of what the bug is.
If you believe this bug is a security issue, please don't use this template and follow our [security guidelines](/SECURITY.md)
-->
Running vm e2e test on the release-1.1 branch failed with the error `Failed to realize Antrea NetworkPolicy`

**To Reproduce**
<!-- Steps to reproduce the behavior. -->

Trigger running of an vm e2e on new prs that based on Antrea 1.12, then trigger running of an vm e2e from release 1.11.3 pr https://github.com/antrea-io/antrea/pull/5239

**Expected**
<!-- A clear and concise description of what you expected to happen. -->

should pass the tests

**Actual behavior**

Tests will fail with the error log
```
====== Running Antrea e2e Tests for VM ======
2023/07/15 14:56:39 Creating K8s ClientSet
2023/07/15 14:56:39 Collecting information about K8s cluster
2023/07/15 14:56:43 Pod IPv4 network: '192.168.0.0/16'
2023/07/15 14:56:43 Service IPv4 network: '10.96.0.0/12'
2023/07/15 14:56:43 Num nodes: 1
2023/07/15 14:56:43 Applying Antrea YAML
2023/07/15 14:56:46 Waiting for all Antrea DaemonSet Pods
2023/07/15 14:56:47 Checking CoreDNS deployment
=== RUN   TestVMAgent
    fixtures.go:228: Creating 'testvmagent-6kmgv3fb' K8s Namespace
2023/07/15 14:56:47 Applying Antrea YAML
2023/07/15 14:56:50 Waiting for all Antrea DaemonSet Pods
2023/07/15 14:56:51 Checking CoreDNS deployment
    vmagent_test.go:82: List of Windows VMs: 'vmbmtest0-win-0', Linux VMs: 'vmbmtest0-1 vmbmtest0-redhat-0'
    vmagent_test.go:83: Using ServiceAccount vm-agent, Namespace vm-ns
    vmagent_test.go:88: Get info for Linux VM: vmbmtest0-1
    vmagent_test.go:88: Get info for Linux VM: vmbmtest0-redhat-0
    vmagent_test.go:96: Get info for Windows VM: vmbmtest0-win-0
    vmagent_test.go:101: TestVMAgent setup
    vmagent_test.go:262: Stopping antrea-agent on VM: vmbmtest0-1
    vmagent_test.go:104: Creating ExternalNode for VM: vmbmtest0-1
    vmagent_test.go:249: Starting antrea-agent on VM: vmbmtest0-1
    vmagent_test.go:262: Stopping antrea-agent on VM: vmbmtest0-redhat-0
    vmagent_test.go:104: Creating ExternalNode for VM: vmbmtest0-redhat-0
    vmagent_test.go:249: Starting antrea-agent on VM: vmbmtest0-redhat-0
    vmagent_test.go:262: Stopping antrea-agent on VM: vmbmtest0-win-0
    vmagent_test.go:104: Creating ExternalNode for VM: vmbmtest0-win-0
    vmagent_test.go:249: Starting antrea-agent on VM: vmbmtest0-win-0
=== RUN   TestVMAgent/testExternalNode
    vmagent_test.go:195: Running verifyExternalEntityExistence
    vmagent_test.go:148: Verifying ExternalEntity vmbmtest0-1-e8be5, expectExists true
    vmagent_test.go:197: Running verifyExternalNodeRealization
    vmagent_test.go:179: Verify host interface configuration for VM: vmbmtest0-1
    vmagent_test.go:179: Verify host interface configuration for VM: vmbmtest0-1
    vmagent_test.go:195: Running verifyExternalEntityExistence
    vmagent_test.go:148: Verifying ExternalEntity vmbmtest0-redhat-0-3e616, expectExists true
    vmagent_test.go:197: Running verifyExternalNodeRealization
    vmagent_test.go:179: Verify host interface configuration for VM: vmbmtest0-redhat-0
    vmagent_test.go:195: Running verifyExternalEntityExistence
    vmagent_test.go:148: Verifying ExternalEntity vmbmtest0-win-0-b24a8, expectExists true
    vmagent_test.go:197: Running verifyExternalNodeRealization
    vmagent_test.go:179: Verify host interface configuration for VM: vmbmtest0-win-0
=== RUN   TestVMAgent/testExternalNodeWithANP
=== RUN   TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM
=== RUN   TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithTCP
    vmagent_test.go:548: Run iperf3 server on VM vmbmtest0-1 with command iperf3 -s -D -p 5201
I0715 14:57:16.768343    6738 k8s_util.go:996] Creating/updating Antrea NetworkPolicy vm-ns/anp-vmagent-ingress-tcp-entity
    vmagent_test.go:473: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:473
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:424
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:324
        	Error:      	Expected nil, but got: &errors.StatusError{ErrStatus:v1.Status{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ListMeta:v1.ListMeta{SelfLink:"", ResourceVersion:"", Continue:"", RemainingItemCount:(*int64)(nil)}, Status:"Failure", Message:"Internal error occurred: failed calling webhook \"anpmutator.antrea.io\": failed to call webhook: Post \"[https://antrea.kube-system.svc:443/mutate/anp?timeout=5s\](https://antrea.kube-system.svc/mutate/anp?timeout=5s\)": tls: failed to verify certificate: x509: certificate signed by unknown authority", Reason:"InternalError", Details:(*v1.StatusDetails)(0xc000ad6a20), Code:500}}
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithTCP
        	Messages:   	Failed to create Antrea NetworkPolicy
    antreapolicy_test.go:4594: Waiting for ANP '/' to be realized
    vmagent_test.go:474: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:474
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:424
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:324
        	Error:      	Expected nil, but got: &errors.errorString{s:"error when waiting for ANP '/' to be realized: resource name may not be empty"}
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithTCP
        	Messages:   	Failed to realize Antrea NetworkPolicy
    vmagent_test.go:427: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:427
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:324
        	Error:      	Received unexpected error:
        	            	timed out waiting for the condition
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithTCP
        	Messages:   	Failed to verify connectivity after applying ANP
I0715 14:57:43.721904    6738 k8s_util.go:1020] Deleting Antrea NetworkPolicy '/'
    vmagent_test.go:430: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:430
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:324
        	Error:      	Received unexpected error:
        	            	resource name may not be empty
        	            	unable to delete Antrea NetworkPolicy 
        	            	antrea.io/antrea/test/e2e.(*TestData).DeleteANP
        	            		/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/k8s_util.go:1023
        	            	antrea.io/antrea/test/e2e.testANPProtocolTCPOrUDP
        	            		/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:429
        	            	antrea.io/antrea/test/e2e.testANPOnVMs.func1
        	            		/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:324
        	            	testing.tRunner
        	            		/usr/local/go/src/testing/testing.go:1446
        	            	runtime.goexit
        	            		/usr/local/go/src/runtime/asm_amd64.s:1594
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithTCP
        	Messages:   	Failed to remove ANP anp-vmagent-ingress-tcp-entity
    vmagent_test.go:561: Stopped iperf3 on VM vmbmtest0-1
=== RUN   TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithUDP
    vmagent_test.go:548: Run iperf3 server on VM vmbmtest0-1 with command iperf3 -s -D -p 5201
I0715 14:57:47.258548    6738 k8s_util.go:996] Creating/updating Antrea NetworkPolicy vm-ns/anp-vmagent-ingress-udp-entity
    vmagent_test.go:473: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:473
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:424
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:330
        	Error:      	Expected nil, but got: &errors.StatusError{ErrStatus:v1.Status{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ListMeta:v1.ListMeta{SelfLink:"", ResourceVersion:"", Continue:"", RemainingItemCount:(*int64)(nil)}, Status:"Failure", Message:"Internal error occurred: failed calling webhook \"anpmutator.antrea.io\": failed to call webhook: Post \"[https://antrea.kube-system.svc:443/mutate/anp?timeout=5s\](https://antrea.kube-system.svc/mutate/anp?timeout=5s\)": tls: failed to verify certificate: x509: certificate signed by unknown authority", Reason:"InternalError", Details:(*v1.StatusDetails)(0xc0002ec120), Code:500}}
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithUDP
        	Messages:   	Failed to create Antrea NetworkPolicy
    antreapolicy_test.go:4594: Waiting for ANP '/' to be realized
    vmagent_test.go:474: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:474
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:424
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:330
        	Error:      	Expected nil, but got: &errors.errorString{s:"error when waiting for ANP '/' to be realized: resource name may not be empty"}
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithUDP
        	Messages:   	Failed to realize Antrea NetworkPolicy
    vmagent_test.go:427: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:427
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:330
        	Error:      	Received unexpected error:
        	            	timed out waiting for the condition
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithUDP
        	Messages:   	Failed to verify connectivity after applying ANP
I0715 14:58:14.173780    6738 k8s_util.go:1020] Deleting Antrea NetworkPolicy '/'
    vmagent_test.go:430: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:430
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:330
        	Error:      	Received unexpected error:
        	            	resource name may not be empty
        	            	unable to delete Antrea NetworkPolicy 
        	            	antrea.io/antrea/test/e2e.(*TestData).DeleteANP
        	            		/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/k8s_util.go:1023
        	            	antrea.io/antrea/test/e2e.testANPProtocolTCPOrUDP
        	            		/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:429
        	            	antrea.io/antrea/test/e2e.testANPOnVMs.func2
        	            		/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:330
        	            	testing.tRunner
        	            		/usr/local/go/src/testing/testing.go:1446
        	            	runtime.goexit
        	            		/usr/local/go/src/runtime/asm_amd64.s:1594
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithUDP
        	Messages:   	Failed to remove ANP anp-vmagent-ingress-udp-entity
    vmagent_test.go:561: Stopped iperf3 on VM vmbmtest0-1
=== RUN   TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithICMP
I0715 14:58:19.467013    6738 k8s_util.go:996] Creating/updating Antrea NetworkPolicy vm-ns/anp-vmagent-ingress-icmp-ip
    vmagent_test.go:473: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:473
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:396
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:334
        	Error:      	Expected nil, but got: &errors.StatusError{ErrStatus:v1.Status{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ListMeta:v1.ListMeta{SelfLink:"", ResourceVersion:"", Continue:"", RemainingItemCount:(*int64)(nil)}, Status:"Failure", Message:"Internal error occurred: failed calling webhook \"anpmutator.antrea.io\": failed to call webhook: Post \"[https://antrea.kube-system.svc:443/mutate/anp?timeout=5s\](https://antrea.kube-system.svc/mutate/anp?timeout=5s\)": tls: failed to verify certificate: x509: certificate signed by unknown authority", Reason:"InternalError", Details:(*v1.StatusDetails)(0xc000752c00), Code:500}}
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithICMP
        	Messages:   	Failed to create Antrea NetworkPolicy
    antreapolicy_test.go:4594: Waiting for ANP '/' to be realized
    vmagent_test.go:474: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:474
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:396
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:334
        	Error:      	Expected nil, but got: &errors.errorString{s:"error when waiting for ANP '/' to be realized: resource name may not be empty"}
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithICMP
        	Messages:   	Failed to realize Antrea NetworkPolicy
    vmagent_test.go:399: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:399
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:334
        	Error:      	Received unexpected error:
        	            	timed out waiting for the condition
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithICMP
        	Messages:   	Failed to verify connectivity after applying ANP
I0715 14:58:53.355316    6738 k8s_util.go:1020] Deleting Antrea NetworkPolicy '/'
    vmagent_test.go:402: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:402
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:334
        	Error:      	Received unexpected error:
        	            	resource name may not be empty
        	            	unable to delete Antrea NetworkPolicy 
        	            	antrea.io/antrea/test/e2e.(*TestData).DeleteANP
        	            		/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/k8s_util.go:1023
        	            	antrea.io/antrea/test/e2e.testANPProtocolICMP
        	            		/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:401
        	            	antrea.io/antrea/test/e2e.testANPOnVMs.func3
        	            		/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:334
        	            	testing.tRunner
        	            		/usr/local/go/src/testing/testing.go:1446
        	            	runtime.goexit
        	            		/usr/local/go/src/runtime/asm_amd64.s:1594
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithICMP
        	Messages:   	Failed to remove ANP anp-vmagent-ingress-icmp-ip
=== RUN   TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithFQDN
I0715 14:58:55.280645    6738 k8s_util.go:996] Creating/updating Antrea NetworkPolicy vm-ns/anp-vmagent-fqdn
    vmagent_test.go:502: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:502
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:375
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:338
        	Error:      	Received unexpected error:
        	            	Internal error occurred: failed calling webhook "anpmutator.antrea.io": failed to call webhook: Post "[https://antrea.kube-system.svc:443/mutate/anp?timeout=5s](https://antrea.kube-system.svc/mutate/anp?timeout=5s)": tls: failed to verify certificate: x509: certificate signed by unknown authority
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithFQDN
        	Messages:   	Failed to create Antrea NetworkPolicy
=== RUN   TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM
=== RUN   TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithTCP
    vmagent_test.go:548: Run iperf3 server on VM vmbmtest0-win-0 with command cmd.exe /c "iperf3.exe -s -D -p 5201"
I0715 14:58:59.118200    6738 k8s_util.go:996] Creating/updating Antrea NetworkPolicy vm-ns/anp-vmagent-ingress-tcp-entity
    vmagent_test.go:473: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:473
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:424
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:324
        	Error:      	Expected nil, but got: &errors.StatusError{ErrStatus:v1.Status{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ListMeta:v1.ListMeta{SelfLink:"", ResourceVersion:"", Continue:"", RemainingItemCount:(*int64)(nil)}, Status:"Failure", Message:"Internal error occurred: failed calling webhook \"anpmutator.antrea.io\": failed to call webhook: Post \"[https://antrea.kube-system.svc:443/mutate/anp?timeout=5s\](https://antrea.kube-system.svc/mutate/anp?timeout=5s\)": tls: failed to verify certificate: x509: certificate signed by unknown authority", Reason:"InternalError", Details:(*v1.StatusDetails)(0xc000831020), Code:500}}
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithTCP
        	Messages:   	Failed to create Antrea NetworkPolicy
    antreapolicy_test.go:4594: Waiting for ANP '/' to be realized
    vmagent_test.go:474: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:474
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:424
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:324
        	Error:      	Expected nil, but got: &errors.errorString{s:"error when waiting for ANP '/' to be realized: resource name may not be empty"}
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithTCP
        	Messages:   	Failed to realize Antrea NetworkPolicy
    vmagent_test.go:427: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:427
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:324
        	Error:      	Received unexpected error:
        	            	timed out waiting for the condition
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithTCP
        	Messages:   	Failed to verify connectivity after applying ANP
I0715 14:59:27.037568    6738 k8s_util.go:1020] Deleting Antrea NetworkPolicy '/'
    vmagent_test.go:430: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:430
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:324
        	Error:      	Received unexpected error:
        	            	resource name may not be empty
        	            	unable to delete Antrea NetworkPolicy 
        	            	antrea.io/antrea/test/e2e.(*TestData).DeleteANP
        	            		/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/k8s_util.go:1023
        	            	antrea.io/antrea/test/e2e.testANPProtocolTCPOrUDP
        	            		/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:429
        	            	antrea.io/antrea/test/e2e.testANPOnVMs.func1
        	            		/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:324
        	            	testing.tRunner
        	            		/usr/local/go/src/testing/testing.go:1446
        	            	runtime.goexit
        	            		/usr/local/go/src/runtime/asm_amd64.s:1594
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithTCP
        	Messages:   	Failed to remove ANP anp-vmagent-ingress-tcp-entity
    vmagent_test.go:561: Stopped iperf3 on VM vmbmtest0-win-0
=== RUN   TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithUDP
    vmagent_test.go:548: Run iperf3 server on VM vmbmtest0-win-0 with command cmd.exe /c "iperf3.exe -s -D -p 5201"
I0715 14:59:31.079465    6738 k8s_util.go:996] Creating/updating Antrea NetworkPolicy vm-ns/anp-vmagent-ingress-udp-entity
    vmagent_test.go:473: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:473
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:424
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:330
        	Error:      	Expected nil, but got: &errors.StatusError{ErrStatus:v1.Status{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ListMeta:v1.ListMeta{SelfLink:"", ResourceVersion:"", Continue:"", RemainingItemCount:(*int64)(nil)}, Status:"Failure", Message:"Internal error occurred: failed calling webhook \"anpmutator.antrea.io\": failed to call webhook: Post \"[https://antrea.kube-system.svc:443/mutate/anp?timeout=5s\](https://antrea.kube-system.svc/mutate/anp?timeout=5s\)": tls: failed to verify certificate: x509: certificate signed by unknown authority", Reason:"InternalError", Details:(*v1.StatusDetails)(0xc000ae31a0), Code:500}}
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithUDP
        	Messages:   	Failed to create Antrea NetworkPolicy
    antreapolicy_test.go:4594: Waiting for ANP '/' to be realized
    vmagent_test.go:474: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:474
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:424
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:330
        	Error:      	Expected nil, but got: &errors.errorString{s:"error when waiting for ANP '/' to be realized: resource name may not be empty"}
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithUDP
        	Messages:   	Failed to realize Antrea NetworkPolicy
    vmagent_test.go:427: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:427
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:330
        	Error:      	Received unexpected error:
        	            	timed out waiting for the condition
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithUDP
        	Messages:   	Failed to verify connectivity after applying ANP
I0715 14:59:59.004688    6738 k8s_util.go:1020] Deleting Antrea NetworkPolicy '/'
    vmagent_test.go:430: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:430
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:330
        	Error:      	Received unexpected error:
        	            	resource name may not be empty
        	            	unable to delete Antrea NetworkPolicy 
        	            	antrea.io/antrea/test/e2e.(*TestData).DeleteANP
        	            		/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/k8s_util.go:1023
        	            	antrea.io/antrea/test/e2e.testANPProtocolTCPOrUDP
        	            		/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:429
        	            	antrea.io/antrea/test/e2e.testANPOnVMs.func2
        	            		/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:330
        	            	testing.tRunner
        	            		/usr/local/go/src/testing/testing.go:1446
        	            	runtime.goexit
        	            		/usr/local/go/src/runtime/asm_amd64.s:1594
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithUDP
        	Messages:   	Failed to remove ANP anp-vmagent-ingress-udp-entity
    vmagent_test.go:561: Stopped iperf3 on VM vmbmtest0-win-0
=== RUN   TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithICMP
I0715 15:00:03.818989    6738 k8s_util.go:996] Creating/updating Antrea NetworkPolicy vm-ns/anp-vmagent-ingress-icmp-ip
    vmagent_test.go:473: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:473
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:396
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:334
        	Error:      	Expected nil, but got: &errors.StatusError{ErrStatus:v1.Status{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ListMeta:v1.ListMeta{SelfLink:"", ResourceVersion:"", Continue:"", RemainingItemCount:(*int64)(nil)}, Status:"Failure", Message:"Internal error occurred: failed calling webhook \"anpmutator.antrea.io\": failed to call webhook: Post \"[https://antrea.kube-system.svc:443/mutate/anp?timeout=5s\](https://antrea.kube-system.svc/mutate/anp?timeout=5s\)": tls: failed to verify certificate: x509: certificate signed by unknown authority", Reason:"InternalError", Details:(*v1.StatusDetails)(0xc000967080), Code:500}}
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithICMP
        	Messages:   	Failed to create Antrea NetworkPolicy
    antreapolicy_test.go:4594: Waiting for ANP '/' to be realized
    vmagent_test.go:474: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:474
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:396
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:334
        	Error:      	Expected nil, but got: &errors.errorString{s:"error when waiting for ANP '/' to be realized: resource name may not be empty"}
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithICMP
        	Messages:   	Failed to realize Antrea NetworkPolicy
    vmagent_test.go:399: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:399
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:334
        	Error:      	Received unexpected error:
        	            	timed out waiting for the condition
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithICMP
        	Messages:   	Failed to verify connectivity after applying ANP
I0715 15:00:37.739389    6738 k8s_util.go:1020] Deleting Antrea NetworkPolicy '/'
    vmagent_test.go:402: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:402
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:334
        	Error:      	Received unexpected error:
        	            	resource name may not be empty
        	            	unable to delete Antrea NetworkPolicy 
        	            	antrea.io/antrea/test/e2e.(*TestData).DeleteANP
        	            		/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/k8s_util.go:1023
        	            	antrea.io/antrea/test/e2e.testANPProtocolICMP
        	            		/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:401
        	            	antrea.io/antrea/test/e2e.testANPOnVMs.func3
        	            		/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:334
        	            	testing.tRunner
        	            		/usr/local/go/src/testing/testing.go:1446
        	            	runtime.goexit
        	            		/usr/local/go/src/runtime/asm_amd64.s:1594
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithICMP
        	Messages:   	Failed to remove ANP anp-vmagent-ingress-icmp-ip
=== RUN   TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithFQDN
I0715 15:00:38.449513    6738 k8s_util.go:996] Creating/updating Antrea NetworkPolicy vm-ns/anp-vmagent-fqdn
    vmagent_test.go:502: 
        	Error Trace:	/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:502
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:375
        	            				/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/test/e2e/vmagent_test.go:338
        	Error:      	Received unexpected error:
        	            	Internal error occurred: failed calling webhook "anpmutator.antrea.io": failed to call webhook: Post "[https://antrea.kube-system.svc:443/mutate/anp?timeout=5s](https://antrea.kube-system.svc/mutate/anp?timeout=5s)": tls: failed to verify certificate: x509: certificate signed by unknown authority
        	Test:       	TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithFQDN
        	Messages:   	Failed to create Antrea NetworkPolicy
=== CONT  TestVMAgent
    vmagent_test.go:137: TestVMAgent teardown
    vmagent_test.go:148: Verifying ExternalEntity vmbmtest0-1-e8be5, expectExists false
    vmagent_test.go:148: Verifying ExternalEntity vmbmtest0-1-e8be5, expectExists false
    vmagent_test.go:148: Verifying ExternalEntity vmbmtest0-redhat-0-3e616, expectExists false
    vmagent_test.go:148: Verifying ExternalEntity vmbmtest0-redhat-0-3e616, expectExists false
    vmagent_test.go:148: Verifying ExternalEntity vmbmtest0-win-0-b24a8, expectExists false
    vmagent_test.go:148: Verifying ExternalEntity vmbmtest0-win-0-b24a8, expectExists false
    fixtures.go:294: Exporting test logs to '/var/lib/jenkins/workspace/antrea-vm-e2e-for-pull-request/antrea-test-logs/TestVMAgent/beforeTeardown.Jul15-15-01-12'
    fixtures.go:412: Exporting logs from vmbmtest0-1 vmbmtest0-redhat-0
    fixtures.go:415: Error when exporting antrea-agent logs from vmbmtest0-1 vmbmtest0-redhat-0: unable to find 'HostName' for 'vmbmtest0-1 vmbmtest0-redhat-0' in SSH config
    fixtures.go:412: Exporting logs from vmbmtest0-win-0
    fixtures.go:465: Deleting 'testvmagent-6kmgv3fb' K8s Namespace
I0715 15:01:15.978534    6738 framework.go:682] Deleting Namespace testvmagent-6kmgv3fb took 5.380218ms
--- FAIL: TestVMAgent (268.58s)
    --- PASS: TestVMAgent/testExternalNode (15.16s)
    --- FAIL: TestVMAgent/testExternalNodeWithANP (204.52s)
        --- FAIL: TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM (101.35s)
            --- FAIL: TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithTCP (30.33s)
            --- FAIL: TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithUDP (30.61s)
            --- FAIL: TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithICMP (38.47s)
            --- FAIL: TestVMAgent/testExternalNodeWithANP/testANPOnLinuxVM/testANPOnExternalNodeWithFQDN (1.94s)
        --- FAIL: TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM (103.17s)
            --- FAIL: TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithTCP (31.96s)
            --- FAIL: TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithUDP (31.98s)
            --- FAIL: TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithICMP (38.51s)
            --- FAIL: TestVMAgent/testExternalNodeWithANP/testANPOnWindowsVM/testANPOnExternalNodeWithFQDN (0.72s)
FAIL
FAIL	antrea.io/antrea/test/e2e	276.483s
FAIL
```

**Versions:**
<!--
Please provide the following information:
 - Antrea version (Docker image tag).
 - Kubernetes version (use `kubectl version`). If your Kubernetes components have [different versions](https://kubernetes.io/docs/setup/release/version-skew-policy/), please provide the version for all of them.
 - Container runtime: which runtime are you using (e.g. containerd, cri-o, docker) and which version are you using?
 - Linux kernel version on the Kubernetes Nodes (`uname -r`).
 - If you chose to compile the Open vSwitch kernel module manually instead of using the kernel module built into the Linux kernel, which version of the OVS kernel module are you using? Include the output of `modinfo openvswitch` for the Kubernetes Nodes.
-->

kubernetes v1.27.2
Antrea 1.12

**Additional context**
<!-- Add any other context about the problem here, such as Antrea logs, kubelet logs, etc. -->

<!-- (Please consider pasting long output into a [GitHub gist](https://gist.github.com) or any other pastebin.) -->

--- a/ci/jenkins/test-vm.sh
+++ b/ci/jenkins/test-vm.sh
@@ -140,6 +140,10 @@ function apply_antrea {
             exit 1
         fi
     fi
+    TEMP_ANTREA_TAR="antrea-image.tar"
+    docker save antrea/antrea-ubuntu:latest -o $TEMP_ANTREA_TAR
+    ctr -n k8s.io image import $TEMP_ANTREA_TAR
+    rm $TEMP_ANTREA_TAR
     echo "====== Applying Antrea yaml ======"
     ./hack/generate-manifest.sh --feature-gates ExternalNode=true,SupportBundleCollection=true --extra-helm-values "controller.apiNodePort=32767" > ${WORKDIR}/antrea.yml
     kubectl apply -f ${WORKDIR}/antrea.yml
@@ -293,7 +297,7 @@ function run_e2e_vms {
     tar -zcf antrea-test-logs.tar.gz antrea-test-logs
 }
 
-function deliver_antrea_vm {
+function build_antrea_binary {
     export_govc_env_var
     clean_vm_agent ${LIN_HOSTNAMES[@]}
     clean_vm_agent ${WIN_HOSTNAMES[@]}
@@ -303,12 +307,8 @@ function deliver_antrea_vm {
     export GOROOT=/usr/local/go
     export GOCACHE=${WORKSPACE}/../gocache
     export PATH=${GOROOT}/bin:$PATH
-    TEMP_ANTREA_TAR="antrea-image.tar"
 
     make docker-bin
-    docker save antrea/antrea-ubuntu:latest -o $TEMP_ANTREA_TAR
-    ctr -n k8s.io image import $TEMP_ANTREA_TAR
-    rm $TEMP_ANTREA_TAR
     make docker-windows-bin
 
     cp ./build/yamls/externalnode/conf/antrea-agent.conf ${WORKDIR}/antrea-agent.conf
@@ -317,7 +317,7 @@ function deliver_antrea_vm {
 trap clean_antrea EXIT
 fetch_vm_ip
 apply_antrea
-deliver_antrea_vm
+build_antrea_binary
 run_e2e_vms
 
 if [[ ${TEST_FAILURE} == true ]]; then

